Tasks/JenkinsQueueJobV2/job.ts

```diff
@@ -424,8 +424,17 @@ export class Job {
             } else if (httpResponse.statusCode == 404) {
                 // got here too fast, stream not yet available, try again in the future
                 thisJob.stopWork(thisJob.queue.TaskOptions.pollIntervalMillis, thisJob.State);
-            } else if (httpResponse.statusCode != 200) {
+            } else if (httpResponse.statusCode == 401) {
+                    Util.failReturnCode(httpResponse, 'Job progress tracking failed to read job progress');
```

Comment:
Does this message need to be localized?

Reply:
The others were not, but I'll add this to the localized strings. They should probably be localized as well.